stable/prometheus/values.yaml

```diff
@@ -328,6 +328,11 @@ server:
     #     hosts:
     #       - prometheus.domain.com
 
+  ## Node tolerations for server scheduling to nodes with taints
+  ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
+  ##
+  tolerations: []
```

Comment:
Apologies for not explaining clearly the first time, i meant the commented defaults which might also explain about possible combinations the user can choose from. 
```
tolerations: []
  #- key: "key"
  #operator: "Equal|Exists"
  #value: "value"
  #effect: "NoSchedule|PreferNoSchedule|NoExecute(1.6 only)"
```
Could you update as above itself? Also while you are adding toleration for this if you could add same example to `node-exporter ` that would be great.  Otherwise LGTM from myside